pkgs/by-name/lo/localsend/package.nix

```diff
@@ -40,10 +70,25 @@ let
     nativeBuildInputs = [
       pkg-config
       copyDesktopItems
+      moreutils
+      jq
+      cargo
+      rustc
+      rustPlatform.cargoSetupHook
     ];
 
     buildInputs = [ libayatana-appindicator ];
 
+    postPatch = ''
+      cp ${./Cargo.lock} ./Cargo.lock
```

Comment:
Is this file copied from https://github.com/Tienisto/rhttp/blob/main/rhttp/rust/Cargo.lock?